### DISCUSSION TITLE
Snapshots in BORIS

### DISCUSSION THREAD

**Discussion by kvolz99** [olivierfriard/BORIS#441] [Q&A]
Is there a way to change where a snapshot taken of a video frame in BORIS is saved from the source of the media file to a separate folder? I'm pulling my videos from an external hard drive that was set up using Windows but I'm using MacOS so I'm unable to add snapshots directly to the video file location.

**Comment by olivierfriard** [comment]
Hi,
please open a feature request on:
https://github.com/olivierfriard/BORIS/issues/new?assignees=&labels=feature+request&template=feature_request.md&title=
